### PR TITLE
Retirement of some connectors

### PR DESCRIPTION
**Pull request by VivekKumarG**
List of retiring connectors

### CODE REVIEW

**Comment by VivekKumarG** [comment]
@dstrome @JohanFreelancer9 Could you please help me with reviewing and merging this PR. This is urgent to announce deprecation of some apps.

File: Teams/Office-365-custom-connectors.md [hunk 1]
```diff
@@ -29,6 +29,54 @@ Microsoft 365 connectors are used with both Microsoft Teams and Microsoft 365 gr
...
+>SHAREPOINTNEWS
+>SIMPLEINOUT
+>STATUSPAGEIO
+>SUBVERSION
+>TEAMFOUNDATIONSERVER
+>TESTFAIRY
+>TRAVISCI
+>UPDOWN
+>USERLIKE
+>XPDEV
```
Comment (JohanFreelancer9) [inline Teams/Office-365-custom-connectors.md:77]:
```suggestion
> [!IMPORTANT]
> Starting January 2023, some Microsoft 365 connectors will be deprecated. The list of deprecating connectors is as follows:

> - AHA
> - AIRBRAKE
> - AIRCALL
> - APPLINKS
> - APPSIGNAL
> - BEANSTALK
> - BITBUCKET
> - BITBUCKETSERVER
> - BUDDY
> - BUGSNAG
> - BUILDKITE
> - CATSONE
> - CHATRA
> - CIRCLECI
> - CODESHIP
> - GETRESPONSE
> - GHOSTINSPECTOR
> - GROOVE
> - HEROKU
> - HONEYBADGER
> - INTERCOM
> - LOGENTRIES
> - NEWRELIC
> - OPSGENIE
> - PAGERDUTY
> - PAPERTRAIL
> - PINGDOM
> - PIVOTALTRACKER
> - RAYGUN
> - ROLLBAR
> - RUNSCOPE
> - SATISMETER
> - SEMAPHORE
> - SENTRY
> - SHAREPOINTNEWS
> - SIMPLEINOUT
> - STATUSPAGEIO
> - SUBVERSION
> - TEAMFOUNDATIONSERVER
> - TESTFAIRY
> - TRAVISCI
> - UPDOWN
> - USERLIKE
> - XPDEV
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by VivekKumarG** [comment]
Looks good @JohanFreelancer9 
Request you to merge it.

**Comment by JohanFreelancer9** [comment]
> Looks good @JohanFreelancer9 Request you to merge it.

@VivekKumarG, you need to commit my suggested changes, please.

**Comment by VivekKumarG** [comment]
@JohanFreelancer9 - Done. 
I missed that earlier. Thanks!

File: Teams/Office-365-custom-connectors.md [hunk 2]
```diff
@@ -29,6 +29,55 @@ Microsoft 365 connectors are used with both Microsoft Teams and Microsoft 365 gr
 
 Any member of a team can connect their team to popular cloud services with the connectors if the team permissions allow, and all team members are notified of activities from that service. Connectors continue to work after the member who initially set up the connector leaves. Any team member with the permissions to add or remove can modify connectors setup by other members.
 
+> [!IMPORTANT]
+> Starting January 2023, some Microsoft 365 connectors will be deprecated. The list of connectors being deprecated is as follows:
+
```
Comment (JohanFreelancer9) [inline Teams/Office-365-custom-connectors.md:34]:
```suggestion
>
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @VivekKumarG, one more minor change, please.

**Comment by VivekKumarG** [comment]
> Apologies @VivekKumarG, one more minor change, please.

Done :)

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@ashishguptaiitb 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by ashishguptaiitb** [comment]
Hi @scanum , to expediate the doc updates and combine it with other updates, I've moved the updates to another PR (https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness-pr/pull/10956). 
Thanks @VivekKumarG for the updates + a follow-up discussion today.

**Comment by ashishguptaiitb** [comment]
Sorry @JohanFreelancer9 , I missed thanking you for your review. We appreciate a quick review that you did. Thank you :-)

**Comment by JohanFreelancer9** [comment]
> Sorry @JohanFreelancer9 , I missed thanking you for your review. We appreciate a quick review that you did. Thank you :-)

@ashishguptaiitb, no problem at all. It is always a pleasure :-)

**Pull request closed without merging**